echo/README.md

```diff
@@ -0,0 +1,128 @@
+# Middleware-echoweb
```

Comment:
Describe the functionality provided by this library at the beginning of the README